sympy/utilities/iterables.py

```diff
@@ -476,19 +476,19 @@ def variations(seq, n, repetition=False):
 
 
 def subsets(seq, k=None, repetition=False):
-    """Generates all k-subsets (combinations) from an n-element set, seq.
+    """Generates all `k`-subsets (combinations) from an `n`-element set, ``seq``.
 
-       A k-subset of an n-element set is any subset of length exactly k. The
-       number of k-subsets of an n-element set is given by binomial(n, k),
-       whereas there are 2**n subsets all together. If k is None then all
-       2**n subsets will be returned from shortest to longest.
+       A `k`-subset of an `n`-element set is any subset of length exactly `k`. The
+       number of `k`-subsets of an `n`-element set is given by ``binomial(n, k)`` `= \binom{n}{k}`,
+       whereas there are `2^n` subsets all together. If `k` is ``None`` then all
+       `2^n` subsets will be returned from shortest to longest.
 
        Examples
        ========
 
        >>> from sympy.utilities.iterables import subsets
 
-       subsets(seq, k) will return the n!/k!/(n - k)! k-subsets (combinations)
+       ``subsets(seq, k)`` will return the `\frac{n!}{k!(n - k)!}` `k`-subsets (combinations)
```

Comment:
maybe it doesn't like formatted text in the examples region when it is parsing for output?